src/performance_profiles.jl

```diff
@@ -13,21 +13,22 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
 
   (np, ns) = size(T);       # Number of problems and number of solvers.
 
-  T[isinf(T)] = NaN;
-  T[T .< 0] = NaN;
+  T[isinf.(T)] = Inf;
+  T[T .< 0] = Inf;
   minperf = minimum(T, 2);  # Minimal (i.e., best) performance per solver
 
   # Compute ratios and divide by smallest element in each row.
   r = zeros(np, ns);
   for p = 1 : np
     r[p, :] = T[p, :] / minperf[p];
   end
+  r[isinf.(r)]=NaN
 
-  logscale && (r = log2(r));
-  max_ratio = maximum(r);
+  logscale && (r = log2.(r));
+  max_ratio = findmax(r)[1]
```

Comment:
In Julia 0.5.2 `maximum([2.0, NaN])` returned 2.0 and  ` findmax([2.0, NaN])[1]` returned 2.0 as well. But in Julia 0.6.0 `maximum([2.0, NaN])` returns NaN and  ` findmax([2.0, NaN])[1]` returns 2.0. That's why I changed it.